src/utils/staking.js

```diff
@@ -486,10 +489,13 @@ export class Staking {
     Helpers
     ********************************/
 
-    async getContractInstance(contractId, methods) {
+    async getContractInstance(contractId, methods, account) {
```

Comment:
@mattlockyer seems like best for us to remove this method altogether, as most use cases should use `.getAccount()` from wallet (otherwise would fail with 2FA / lockup)

Reply:
what I mean is `await (await new nearApiJs.Account(this.wallet.connection, contractId)).state()` is completely redundant

Reply:
it would be basically: 

```
const contract = new Contract(await this.wallet.getAccount(accountId), contractId, contractMethods)
```

Reply:
I don't know how we would get the contract instance of validators then?
we need to calc fee percentage from contract
```
const fee = validator.fee = await validator.contract.get_reward_fee_fraction()
```

Reply:
@mattlockyer I meant we might just inline it when making a call, as you anyway are passing different `account` instance.

but also you generally can do `await account.viewFunction(validator.accountId,  'get_reward_fee_fraction')`  in cases when you only need a one off